### DISCUSSION TITLE
How to use webhook with Azure DevOps ?

### DISCUSSION THREAD

**Discussion by monaka** [argoproj/argo-cd#8690] [Q&A] [answered]
Hello,

I'm using ArgoCD with git repo on Azure DevOps.
And I want to use the git webhook provided by ArgoCD.
But it seems ArgoCD only supports GitHub|BitBucket|GitLab|BitBuckerServer|Gogs.
https://argo-cd.readthedocs.io/en/stable/operator-manual/webhook/

Are there workarounds to use ?

**Comment by todaywasawesome** [comment, marked as answer]
Check out this discussion about using Azure DevOps as the repo with Argo CD https://github.com/argoproj/argo-cd/issues/6362

A webhook can increase the speed of syncing but isn't required to actually sync and keep things in sync. By default Argo CD will check repos every three minutes.

**Comment by doug62** [comment]
Has any progress been made on this critical feature?
